docs/api/filters.rst

```diff
@@ -10,4 +15,111 @@ Filters
+
+.. note:: ``histolab`` stores masks as NumPy arrays. The utility class ``ToPILImage`` in the image filters module retrieves the Pillow Image from the corresponding array.
+
+* Morphological filters:
+    * **Image preprocessing**: Morphology is a comprehensive set of image processing operations that transform images by using geometrical structures. Morphological operations act by adjusting image pixels based on the value of the pixels in the neighborhood. The choice of the neighborhood's size and shape will affect the behaviour of the morphological operation so that it will be sensitive to specific shapes in the input image [9]_. The *structuring element* is the component of the morphological operations that defines the considered neighborhood: it is a shape (typically a circle or a square) that determines the area used to process each pixel in the image. Usually, the shape and size of the structuring element are chosen to reflect the geometry of the objects in the image that the structuring element will transform: for example, linear structuring element would be used to detect lines in an image. Morphological operations can be applied to binary masks to shrink or enlarge regions of the image. Classic morphological operations include dilation, erosion, opening and closing; ``histolab`` implements these operations in the filters submodule ``morphological_filters``. The default structuring element is a disk, with radius 5 for both dilation and erosion, and radius 3 for both opening and closing. However, it is possible to override the default value by passing ``disk_size=N`` as parameter to the filter constructor. The ``morphological_filters`` module implements three additional morphological operations useful for manipulating binary masks: `WhiteTopHat <filters/morphological_filters.html#src.histolab.filters.morphological_filters.WhiteTopHat>`_, `RemoveSmallObjects <filters/morphological_filters.html#src.histolab.filters.morphological_filters.RemoveSmallObjects>`_, and `RemoveSmallHoles <filters/morphological_filters.html#src.histolab.filters.morphological_filters.RemoveSmallHoles>`_. The *white top-hat* transformation is defined as the difference between the image and its morphological opening with respect to a structuring element. This operation results in an image including only structures smaller than the structuring element and brighter than their neighborhood, and it is thus used to extract light details on a dark background. The white top-hat filter uses a cross-shaped structuring element with connectivity 1 by default. The ``RemoveSmallObjects`` filter removes objects with an area smaller than a specified value while the ``RemoveSmallHoles`` filter "fills" holes with an area smaller than the specified threshold. The minimal area value is set to 3000 for both filters.
```

Comment:
please add link to "morphological_filters", "RemoveSmallObjects", and "RemoveSmallHoles"